dali/operators/image/remap/cvcuda/warp_perspective.cc

```diff
@@ -38,8 +38,8 @@ Performs a perspective transform on the images.
     .NumInput(1, 2)
     .InputDox(0, "input", "TensorList of uint8, uint16, int16 or float",
               "Input data. Must be images in HWC or CHW layout, or a sequence of those.")
-    .InputDox(1, "matrix_gpu", "1D TensorList of float",
-              "Transformation matrix data. Should be used to pass the GPU data. "
+    .InputDox(1, "matrix_gpu", "2D TensorList of float",
+              "3x3 Perspective transform matrix. Should be used to pass the GPU data. "
```

Comment:
```suggestion
              "3x3 Perspective transform matrix. Should be used to pass the GPU data. "
```
1. It can be a flattened 1D TensorList, AFAIR
2. It's not 3x3 - it's either 3x2 (for 2D data) or 4x3 (for 3D data). We're not using homogeneous coordinates here.